docs/src/upkeep/images.md

```diff
@@ -1,146 +1,37 @@
-
-Verify that the root filesystem list is correct:
-
-```
-$ make rootfs-all-print
-void-i686-ROOTFS-20190217.tar.xz
-void-x86_64-ROOTFS-20190217.tar.xz
-void-x86_64-musl-ROOTFS-20190217.tar.xz
-void-armv6l-ROOTFS-20190217.tar.xz
-void-armv6l-musl-ROOTFS-20190217.tar.xz
-void-armv7l-ROOTFS-20190217.tar.xz
-void-armv7l-musl-ROOTFS-20190217.tar.xz
-void-aarch64-ROOTFS-20190217.tar.xz
-void-aarch64-musl-ROOTFS-20190217.tar.xz
-```
-
-Your list should match with the exception of dates.  If it does not,
-either the makefile or the docs are out of date, figure out which one
-and send a patch.
-
-Now you are ready to build the root filesystems:
+The images should be built using Github CI in the void-mklive repository. This
+can be triggered [on
+Github](https://github.com/void-linux/void-mklive/actions/workflows/gen-images.yml)
+or by using the `release.sh` script in void-mklive:
 
 ```
-$ sudo make rootfs-all
+$ ./release.sh start
 ```
 
-Make here should be called with sudo to ensure that sudo doesn't time
-out inside of make.
+By default, this will build:
 
-### Building the platform filesystems.
+- Live ISOs with `base` and `xfce` variants for `x86_64*` and `i686`
+- ROOTFSes for `x86_64*`, `i686`, `aarch64*`, `armv7l*`, and `armv6l*`
+- PLATFORMFSes for `aarch64*`, `armv7l*`, and `armv6l*` Raspberry Pis
+- SBC images for `aarch64*`, `armv7l*`, and `armv6l*` Raspberry Pis
 
-Once the root filesystems have built, they can be specialized into
-platform filesystems.  These have system specific adaptations added to
-them which make them suitable for installation targets.
-
-Verify that the platformfs list is correct:
-
-```
-$ make platformfs-all-print
-void-rpi-PLATFORMFS-20190217.tar.xz
-void-rpi-musl-PLATFORMFS-20190217.tar.xz
-void-rpi2-PLATFORMFS-20190217.tar.xz
-void-rpi2-musl-PLATFORMFS-20190217.tar.xz
-void-rpi3-PLATFORMFS-20190217.tar.xz
-void-rpi3-musl-PLATFORMFS-20190217.tar.xz
-void-beaglebone-PLATFORMFS-20190217.tar.xz
-void-beaglebone-musl-PLATFORMFS-20190217.tar.xz
-void-cubieboard2-PLATFORMFS-20190217.tar.xz
-void-cubieboard2-musl-PLATFORMFS-20190217.tar.xz
-void-odroid-c2-PLATFORMFS-20190217.tar.xz
-void-odroid-c2-musl-PLATFORMFS-20190217.tar.xz
-void-usbarmory-PLATFORMFS-20190217.tar.xz
-void-usbarmory-musl-PLATFORMFS-20190217.tar.xz
-void-GCP-PLATFORMFS-20190217.tar.xz
-void-GCP-musl-PLATFORMFS-20190217.tar.xz
-```
-
-As above your list should match with the exception of the datecodes.
-If it does not match, it is likely this list is out of date and
-another platform has been added or one of the old ones has been
-dropped.  Send a patch to update this page.
-
-As above call make with sudo to build the filesystems:
-
-```
-$ sudo make platformfs-all
-```
+This will take approximately 2 hours for the default settings. To ensure all
+images have the same datecode, the datecode is cached at the beginning of the
+run. The CI workflow will also generate `sha256sum.txt` for the built images.
 
 ### Building the PXE images.
```

Comment:
might be nice to get PXE images in the same workflow as regular images and start publishing them again, but that sounds like a task for the next image release. thus, leaving that section there for now.

Reply:
I would also like to re-enable them, but perhaps using mkinitcpio instead of dracut so they don't need a whole iso image built just to generate the initrd.